### ISSUE TITLE
JSON Editor UI for Study Design is only usable in one view (Firefox)

### ISSUE THREAD

**Issue by ianchandlercampbell** [lookit/lookit-api#1453]
# TL;DR
The JSON editor UI in general does not work well with browser scaling, but one major problem is that only one view selection (left-aligned) allows the user to scroll at least in Firefox.

# Narrative

I would like to request a different JSON editor that:
(1) Scales well with different monitor/browser sizes
(2) does not hide its scroll bar behind the browser scroll bar/ has other window options
(3) Has a pop-out window/separate window for the editor

I would put this on my feature request list for the future jsPsych code editor as well! Given that's the future, this could also be a request only for the jsPsych code editor.

Works:

![image](https://github.com/user-attachments/assets/2bfa20ac-5c2e-458b-938c-513e0f2ee993)

Hidden behind browser scroll:

![image](https://github.com/user-attachments/assets/4430af06-10bc-4958-b21f-5d1dda190530)
